download/Cargo.toml

```diff
@@ -16,11 +16,11 @@ reqwest-backend = ["reqwest", "env_proxy", "lazy_static"]
 
 [dependencies]
 error-chain = "0.12"
-url = "1.7"
+url = "2.1"
 curl = { version = "0.4.11", optional = true }
-env_proxy = { version = "0.3.1", optional = true }
+env_proxy = { version = "0.4", optional = true }
 lazy_static = { version = "1.0", optional = true }
-reqwest = { version = "0.9.21", features = ["socks"], optional = true }
+reqwest = { git = "https://github.com/seanmonstar/reqwest", rev = "7631c0390e90d539b825ed27693a7a0bb1e31dc8", features = ["blocking", "gzip"], optional = true }
 
 [dev-dependencies]
 hyper = "0.13"
```

Comment:
```suggestion
hyper = { version = "0.13", default-features = false, features = ["tcp"] }
```

Reply:
Also tokio:
```diff
-tokio = { version = "0.2" }
+tokio = { version = "0.2.2", default-features = false, features = ["tcp", "sync"] }
```

Reply:
These dependencies are for testing. I seem we don't have to specify them strictly.

Reply:
At least for me it will limit the unused indirection deps what we use.

Reply:
Okay. I fixed them in https://github.com/rust-lang/rustup/pull/2154/commits/0f3af10c8b9b3eacbe9a404edd250d9d15f66343